hvac_services/demo/product_pricelist.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_pricelist_1" model="product.pricelist">
```

Comment:
This one is generated automatically, you can remove (and references to it as well)